src/icons/fileIcons.ts

```diff
@@ -317,15 +317,15 @@ export const fileIcons: FileIcons = {
       name: 'certificate',
       fileExtensions: ['cer', 'cert', 'crt'],
       fileNames: [
+        'copying',
+        'copying.md',
+        'copying.txt',
         'license',
         'license.md',
         'license.txt',
         'licence',
         'licence.md',
-        'licence.txt',
-        'unlicense',
```

Comment:
Why did you remove the unlicense? I think we should leave it here :)

Reply:
The reasoning is in the opening message of the PR:

> Removed the "unlicense" filename, which is specific to a single license, and by itself does not guarantee that the file is actually a license. This reverts #474. Here are some examples of false positives:
>  - [`_licenses/unlicense.txt` at github/choosealicense.com](https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/unlicense.txt)
>  - [`src/.../SPDX/Unlicense` at fossology/fossology](https://github.com/fossology/fossology/blob/master/src/nomos/agent_tests/testdata/NomosTestfiles/SPDX/Unlicense)
>  - [`website/Unlicense` at spdx/license-list-data](https://github.com/spdx/license-list-data/blob/master/website/Unlicense)

Reply:
Got it, but do you think that these false positives occur that often? Not sure about that. It's the same for `copying.txt`. It might happen that someone is filling it with other content than the license context.

Reply:
The difference is that "COPYING" is a common filename for license files, recommended explicitly by well-known licenses such as those from the GNU project, and by extension other licenses picked it up (even when it's not explicitly recommended by the license's steward; for example, [here](https://github.com/nazar-pc/PickMeUp/blob/master/copying.md)'s a 0BSD-licensed project that independently chose to name the file "copying" despite that license's instructions not requiring or recommending it).

This common practice is evidenced in Choose A License's [guidelines](https://choosealicense.com/community/):

> If you’re contributing to or extending an existing project, it’s almost always easiest to continue using that project’s license. To find its license, look for a file called LICENSE or COPYING, and skim the project’s README.

All that said, I am not strongly advocating for removing the "unlicense" filenames from this list. I only suggested that because it seemed odd to target a specific license in a list of generic filenames. I can re-add them if you prefer.

Reply:
Yep, you're right! Thanks for the explanation :)

Reply:
No problem! I appreciate your care with ensuring the changes make sense. And this way the detailed explanation is recorded for posterity ;)